### DISCUSSION TITLE
tableの隔行にあった背景色の塗り分けがなくなった。

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#806] [要望・アイデア・不具合]
ここ１年間の「Qiitaアップデートのお知らせ」を読み返してみても、どの時点で変更されたのかがはっきりしません。いつの間にか、全行均一の白一色の背景に変わっています。

　塗り分けにも一長一短あるので、均一でも困ることはありませんが、単純に塗り分けをやめただけの状態では、罫線が淡くて見え難く、**テーブルとしての存在感が弱すぎます**。改善を期待します。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご報告ありがとうございます！ご不便をおかけして申し訳ありません。
意図せず塗り分けが無くなっているため、修正いたします🙇

**Reply by kabechiyo** [reply]
@tsubolabo 
おまたせしました！
こちらの不具合の修正が完了しました！
